scripts/variantstore/variant-annotations-table/GvsValidateVAT.wdl

```diff
@@ -1328,10 +1342,11 @@ task SpotCheckForAAChangeAndExonNumberConsistency {
                 echo "The VAT table ~{fq_vat_table} has been spot checked for aa_change and exon_number consistency." > ~{results_file}
             else
                 echo "The VAT table ~{fq_vat_table} has failed the spot check for aa_change and exon_number consistency." > ~{results_file}
+            fi
```

Comment:
I think there should only be one `fi` here; with both this and the one on line 1346 there's going to be a parse error. I would keep this one and get rid of the one on 1346 for indent reasons.